File: core/src/ru/myitschool/dcrawler/math/NumberAction.java
package ru.myitschool.dcrawler.math;

/**
 * Created by Voyager on 04.08.2017.
 */
public class NumberAction extends MathAction {

    private int result;

    public NumberAction(int result) {
        this.result = result;
    }

    @Override
    public int act() {
        return result;
    }

    @Override
    public String getDescription() {
        return "" + result;
    }
}

File: core/src/com/wolg_vlad/dcrawler/math/NumberAction.kt
package com.wolg_vlad.dcrawler.math

/**
 * Created by Voyager on 04.08.2017.
 */
class NumberAction(private val result: Int) : MathAction() {
    override fun act(): Int {
        return result
    }

    override val description: String?
        get() = "" + result

}